packages/shared/src/components/CombatReport/index.tsx

```diff
@@ -83,21 +84,14 @@ export const CombatReport = ({ combat, anon }: IProps) => {
             Replay
           </a>
         </div>
-        {activeTab === 'summary' && (
-          <div className="mt-4 mx-2">
-            <CombatSummary />
+        <div className="mt-4 mx-2 flex-1 relative overflow-y-scroll">
```

Comment:
using mx-2 and padding from the outer container is causing the scrollbar to be offset from the edge of the screen
ultimately not a deal breaker for me but something to consider

![image](https://user-images.githubusercontent.com/15525519/204152424-d1b11b65-0f94-4c6f-840a-ef3751324749.png)

Reply:
oh yes the margin needs to be removed.